service/controller/app/v1/resource_set.go

```diff
@@ -129,7 +106,6 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 	}
 
 	resources := []controller.Resource{
-		configMapResource,
```

Comment:
Is this intentional?

Reply:
Yes, since `configMap` is not ready yet we are hitting error after booting app operator.

Reply:
Yes this was me. 😞 

I'm still working on the desired state for the configmap resource. I thought it would be OK because there is no desired state yet but no it wasn't. 

I'll put the wiring back once the resource is complete.